SpatialGDK/Source/SpatialGDK/Private/Interop/GlobalStateManager.cpp

```diff
@@ -29,6 +29,22 @@ void UGlobalStateManager::Init(USpatialNetDriver* InNetDriver, FTimerManager* In
 	Receiver = InNetDriver->Receiver;
 	TimerManager = InTimerManager;
 	GlobalStateManagerEntityId = SpatialConstants::INITIAL_GLOBAL_STATE_MANAGER_ENTITY_ID;
+
+#if WITH_EDITOR
```

Comment:
With editor just implies and editor build and not actually running PIE right? I think we need to create a more accurate check for running in PIE.
@davedolben was talking about this at one time.

Reply:
this code is intended to fix our issues when running in editor and launching a client through pie.

Reply:
Yeah that's correct. This is just a preprocessor macro that prevents the code from being built into editor builds. It does not actually check for PIE, and if you launch a managed worker via `UE4Editor.exe` but outside the graphical editor (e.g. with `-game` or `-server`), then this code will still be included and run.

I don't remember exactly how to check for PIE, but it's done at least once more somewhere in the GDK codebase.

Reply:
I think this is fine as we just don't want to enable this code in standalone game executables.